config/README.md

```diff
@@ -17,12 +22,47 @@ Copy this file to `~/.aidx/config.json` and update with your values:
 }
 ```
 
+## Required Configuration
+
+### Basic Query Execution
+- `applicationId`: Your Application Insights Application ID
+- `tenantId`: Your Azure Tenant ID
+- `openAI.endpoint`: Your Azure OpenAI endpoint
+- `openAI.deploymentName`: Your OpenAI deployment name (default: gpt-4)
+
+### External Execution (Optional)
```

Comment:
We don’t need these config change because appinsight's resource id is automatically detected, and azure data explorer is out of our scope.